geoip.go

```diff
@@ -338,3 +338,19 @@ func (gi *GeoIP) GetCountry_v6(ip string) (cc string, netmask int) {
 	}
 	return
 }
+
+func (gi *GeoIP) GetDatabaseInfo() (databaseInfos string) {
```

Comment:
I just copied/pasted one of your methods, but I prefer the unnamed return version